### ISSUE TITLE
Unmapped Read/Write error occurs in Xenoblade Chronicles 3

### ISSUE THREAD

**Issue by PlatinumLucario** [yuzu-emu/yuzu#9600]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Mainline 1296, Mainline 1298, Mainline 1299, Mainline 1300, Mainline 1301

### Description of Issue

After rendering 3D objects and loading textures and unloading textures, eventually the log begins to repeatedly output errors such as this:
```
[2757.378327] HW.Memory <Error> core/memory.cpp:operator():643: Unmapped Write32 @ 0x000000000000256C = 0x000000003F800000
[2757.378328] HW.Memory <Error> core/memory.cpp:operator():643: Unmapped Write32 @ 0x000000000000259C = 0x0000000000000000
[2757.378430] HW.Memory <Error> core/memory.cpp:operator():620: Unmapped Read64 @ 0x3F80000000000008
[2757.378432] HW.Memory <Error> core/memory.cpp:operator():620: Unmapped Read64 @ 0x3F80000000000028
[2757.378432] HW.Memory <Error> core/memory.cpp:operator():620: Unmapped Read64 @ 0x3F80000000000028
```

This could be related to what causes random emulator crashes while playing, it seems to be something to do with Hardware Memory (HW.Memory) not being mapped and allocated correctly. I would suggest rechecking the mallocs, callocs and reallocs, to see if they're correctly mapped for each read and write function that the games use.

I will be testing Mainline 1302 later as well, to see if I can reproduce the same bug.

### Expected Behavior

All memory allocations should be fully mapped without any problems.

### Reproduction Steps

1. Open Yuzu.
2. Play Xenoblade Chronicles 3.
3. Move around in the 3D environments in Xenoblade Chronicles 3 for some time.

### Log File

[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/10384573/yuzu_log.txt)

### System Configuration

CPU: Intel® Core™ i3-6100 CPU @ 3.70GHz × 4
GPU: NVIDIA Corporation GM206 [GeForce GTX 950]
GPU Driver: nvidia-driver-525
RAM: 16 GiB
OS: Ubuntu 22.04.1 LTS

**Comment by PlatinumLucario** [comment]
Not at all, just to confirm, all patches and mods are disabled while I was running the game.

Interesting, so it's accessing invalid memory addresses?

**Comment by liamwhite** [comment]
Yes. Are you able to find a yuzu mainline version which does not exhibit these crashes?

**Comment by goldenx86** [comment]
Here, our old mainlines storage: https://github.com/yuzu-emu/yuzu-mainline/releases

**Comment by Chirronupu** [comment]
I'm having similar issues in SSBU, most recent build without error / frequent crashes is mainline 1296.

CPU: Intel Core i7-5775C
GPU: AMD RX 580 8GiB
GPU Driver: 22.11.2
RAM: 16 GiB
OS: Windows 11 22H2

**Comment by Chirronupu** [comment]
I'm having similar issues in SSBU, most recent build without error / crashes is mainline 1295.

CPU: Intel Core i7-5775C
GPU: AMD RX 580 8GB
GPU Driver: 22.11.2
RAM: 16 GB
OS: Windows 11 22H2
